addons/udes_stock_refactoring/models/stock_quant_package.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class StockQuantPackage(models.Model):
+    _inherit = "stock.quant.package"
+
+    def get_package(self, package_identifier, create=False, no_results=False):
```

Comment:
Looks like get_package, get_group are very similar. Might be useful to consider a mixin class for this, like mixin_stock_model in udes_stock.

Reply:
This module should depend on udes_stock which already has a function to do this. Although looks like we only need this function for get_pickings which is not really needed.

Reply:
Yeah this method is no longer needed and has now been removed.